examples/android/WORKSPACE.bazel

```diff
@@ -0,0 +1,50 @@
+workspace(name = "android_examples")
```

Comment:
Does this need to be it's own workspace? I feel we'd get better coverage by having it be a part of `rules_rust_examples`

Reply:
the only annoyance is if you want to run those locally, you have to have the android SDK / NDK setup which I imagine most testers won't, I'm happy to merge them if you want though!

Reply:
I think it can be done at a later date but in general, fewer workspaces would be better